frontend/src/pages/account/Create.vue

```diff
@@ -114,6 +123,14 @@ export default {
         }
     },
     methods: {
+        loadCustomContent () {
+            SettingsAPI.getSettings().then((settings) => {
+                if (settings['branding:account:signUpLeftBanner']) {
+                    console.log(settings['branding:account:signUpLeftBanner'])
+                    this.splash = settings['branding:account:signUpLeftBanner']
+                }
+            })
```

Comment:
You already have `this.settings['branding:account:signUpLeftBanner']` available (note the `mapState` on line 88). No need to call the settings api yourself.